lib/commonFunctions.js

```diff
@@ -19,6 +19,42 @@ var sundial = require('sundial');
 
 var annotate = require('./eventAnnotations');
 
+/**
+ * Computes the number of milliseconds after midnight on the date specified.
+ *
+ * @param dateTime DateTime object to figure out millis from
+ * @returns {number} number of millis in current day
+ */
+exports.computeMillisInCurrentDay = function(e){
+  var msFromMidnight = sundial.getMsFromMidnight(e.time, e.timezoneOffset);
```

Comment:
So I moved this function from the CareLink driver, and while writing the tests just now I started wondering if we shouldn't be using `clockDriftOffset` instead of `conversionOffset` in this function? Or am I misunderstanding how it works?

I've added some tests to `testCommonFunctions.js` - let me know if they're sufficient or not.